php/get-pkce-access-token

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env php
```

Comment:
This file needs execute permission

Reply:
`ls -l` now matches the other file permissions, so this should be fixed in my most recent commit.